pkg/services/ngalert/notifier/channels_config/available_channels.go

```diff
@@ -1708,6 +1709,154 @@ func GetAvailableNotifiers() []*NotifierPlugin {
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					Placeholder:  jira.DefaultSummary,
+					PropertyName: "summary",
+				},
+				{
+					Label:        "Description",
+					Description:  fmt.Sprintf("The description of the Jira issue. You can use templates to customize this field. Maximum length is %d characters. Otheriwse, the text will be truncated", jira.MaxDescriptionLenRunes),
+					Element:      ElementTypeTextArea,
+					InputType:    InputTypeText,
+					Placeholder:  jira.DefaultDescription,
+					PropertyName: "description",
+				},
+				{
+					Label:        "Labels",
+					Description:  "Labels to assign to the Jira issue. You can use templates to customize this field.",
+					Element:      ElementStringArray,
+					Placeholder:  "",
+					PropertyName: "labels",
+				},
+				{
+					Label:        "Priority",
+					Description:  "The priority of the Jira issue (e.g., High, Medium, Low). You can use templates to customize this field.",
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					Placeholder:  jira.DefaultPriority,
+					PropertyName: "priority",
+					Required:     false,
+				},
+				{
+					Label:        "Resolve Transition",
+					Description:  `Name of the workflow transition to resolve an issue. The target status must have the category "done". If not set, the issue will not be resolved.`,
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					Placeholder:  "",
+					PropertyName: "resolve_transition",
+					Required:     false,
+				},
+				{
+					Label:        "Reopen Transition",
+					Description:  `Name of the workflow transition to resolve an issue. The target status must not have the category "done". If not set, the issue will not be reopened.`,
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					Placeholder:  "",
+					PropertyName: "reopen_transition",
+					Required:     false,
+				},
+				{
+					Label:        "Reopen Duration",
+					Description:  "Reopen the issue when it is not older than this value in minutes. Otherwise, create a new issue.",
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					Placeholder:  "10m",
+					PropertyName: "reopen_duration",
+				},
+				{
+					Label:        "\"Won't fix\" Transition",
+					Description:  `If reopen transition is defined, ignore issues with that resolution`,
+					Element:      ElementTypeInput,
+					InputType:    InputTypeText,
+					Placeholder:  "",
+					PropertyName: "wont_fix_resolution",
+					Required:     false,
+				},
+				{
+					Label:          "Custom field ID for deduplication",
+					Description:    "Id of custom field where the deduplication key should be stored. Otherwise, it will be added to labels in format 'ALERT($KEY)'",
```

Comment:
```suggestion
					Description:    "Id of the custom field where the deduplication key should be stored. Otherwise, it is added to labels in format 'ALERT($KEY).'",
```
if possible, docs and UI text should be present tense, not future tense. also, i added a period and preposition.